external/build_image.sh

```diff
@@ -42,7 +42,7 @@ check_external_custom=$9
 if [[ "${check_external_custom}" == "0" ]]; then
 	set_test $1
 fi
-set_version $2
```

Comment:
Was this deliberate?

Reply:
yes, as we don't need the version check.S removed the set_version function.